stack/building-with-pip.rst

```diff
@@ -0,0 +1,241 @@
+###################################
+Making Your Package pip Installable
+###################################
+
+By default a science pipelines package will be configured to work with SCons and EUPS but not with ``pip``.
+If the package does not include C++ code and does not depend on any of the C++ packages, such as ``afw`` or ``daf_base`` then it is possible to make the package installable with ``pip`` as well as with SCons.
+
```

Comment:
When should developers/package owners choose to make their package pip-installable?  What criteria should be considered?  Does the DM-CCB have to approve?

Reply:
This needs some discussion. We might want to say that a package should be pip-installable if it can be pip-installable and it should go on PyPI if it can go there. For science pipelines most packages can't be pip-installable and can't go on PyPI.